metabase/metabase
# Add metabase commercial license to embedding-sdk

### Description

Adds the LICENSE.txt commercial license to the embedding-sdk output bundle, and reference to it in package.json.

### How to verify

Install the package, and see if the LICENSE.txt file is copied to the installed node_modules package.
